src/t_zset.c

```diff
@@ -1976,8 +1976,8 @@ void zremrangeGenericCommand(client *c, zrange_type rangetype) {
         notifyKeyspaceEvent(NOTIFY_ZSET,notify_type,key,c->db->id);
         if (keyremoved)
             notifyKeyspaceEvent(NOTIFY_GENERIC,"del",key,c->db->id);
+        server.dirty += deleted;
```

Comment:
> Move server.dirty += deleted; inside if (deleted. Just like others do. (But maybe not necessary. Can avoid a += 0 call)

No, Our style is put `server.dirty += deleted` In `if (deleted)`, see [t_hash.c#L848](https://github.com/redis/redis/blob/unstable/src/t_hash.c#L848) and [t_set.c#L346](https://github.com/redis/redis/blob/unstable/src/t_set.c#L346).

if your `others` means [t_zset.c#L1980](https://github.com/redis/redis/blob/unstable/src/t_zset.c#L1980), i think we need change L1980, move it to `if (deleted)`.